package.json

```diff
@@ -38,7 +38,7 @@
     "@types/jest": "^27.5.2",
     "@types/node": "^18.0.0",
     "@types/react": "^17.0.2",
-    "@types/styled-components": "^5.1.25",
+    "@types/styled-components": "^5.1.26",
```

Comment:
while we're here, should we consider upgrading to the latest version? i.e. I believe we're on 5.3.6 - from what I can tell there aren't any breaking changes